src/domain/journey/space/SpaceCommunityPage/SpaceCommunityPage.tsx

```diff
@@ -89,6 +96,11 @@ const SpaceCommunityPage = () => {
 
   const [isActivitiesDialogOpen, setIsActivitiesDialogOpen] = useState(false);
 
+  const virtualContributors: VirtualContributorProps[] =
+    data?.space.community.virtualContributors?.filter(vc => vc?.searchVisibility === SearchVisibility.Public) ?? [];
+
+  const hasReadPrivilege = data?.space.authorization?.myPrivileges?.includes(AuthorizationPrivilege.Read);
```

Comment:
I'm not sure whether the effort is worthy performance-wise but in case hasReadPrivilege === false, then there's no need of the assignment and filtering on line 100 (on other occasions as well).
It could be optimized like:

```
let virtualContributors: VirtualContributorProps[] | undefined;

const hasReadPrivilege = data?.space.authorization?.myPrivileges?.includes(AuthorizationPrivilege.Read);

if (hasReadPrivilege) {
 virtualContributors = data?.space.community.virtualContributors?.filter(vc => vc?.searchVisibility === SearchVisibility.Public);
}